.github/workflows/htmlhint_linter.yml

```
@@ -0,0 +1,21 @@
+name: HTML Linter
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  lint:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
```

If there's a reason to keep it on v2, that's fine, but otherwise, let's follow @wknowleskellett's suggestion and upgrade to v3. (This isn't a big deal though.)

```suggestion
    - uses: actions/checkout@v3
    - uses: actions/setup-node@v3
```